src/Service/Cache/UserCacheWarmerService.php

```diff
@@ -30,20 +32,35 @@
 
 class UserCacheWarmerService
 {
-    private const MESSAGE_SIZE = 100;
+    private const MAX_RETRY_COUNT = 5;
 
     private MessageBusInterface $messageBus;
     private LoggerInterface $messengerLogger;
     private LoggerInterface $cacheWarmupLogger;
+    private int $messagePayloadSize;
+    private int $retryWaitInterval;
 
     public function __construct(
         MessageBusInterface $messageBus,
         LoggerInterface $messengerLogger,
-        LoggerInterface $cacheWarmupLogger
+        LoggerInterface $cacheWarmupLogger,
+        int $cacheWarmupPayloadBatchSize,
+        int $userCacheWarmupRetryWaitInterval
     ) {
         $this->messageBus = $messageBus;
         $this->messengerLogger = $messengerLogger;
         $this->cacheWarmupLogger = $cacheWarmupLogger;
+
+        if ($cacheWarmupPayloadBatchSize < 1) {
+            throw new InvalidArgumentException('Message payload size must be greater or equal to 1.');
+        }
+
+        if ($userCacheWarmupRetryWaitInterval < 1) {
```

Comment:
Though it's not a problem at all, but having minimum at 1 microsecond doesn't make sense :upside_down_face:

Reply:
Fair point, fixed :)